src/runtime/native/ABI.cs

```diff
@@ -0,0 +1,32 @@
+namespace Python.Runtime.Native
+{
+    using System;
+    using System.Globalization;
+    using System.Linq;
+    using System.Reflection;
+    using System.Runtime.InteropServices;
+
+    static class ABI
+    {
+        internal static void Initialize(Version version, BorrowedReference pyType)
+        {
+            string offsetsClassSuffix = string.Format(CultureInfo.InvariantCulture,
+                                                      "{0}{1}", version.Major, version.Minor);
+
+            var thisAssembly = Assembly.GetExecutingAssembly();
+
+            const string nativeTypeOffsetClassName = "Python.Runtime.NativeTypeOffset";
```

Comment:
Use `nameof` here.

Reply:
This class does not actually exist, if built without setup.py (which should be supported, once Modernization project is completed).

Reply:
Good point.